api/pom.xml

```diff
@@ -81,13 +81,51 @@
             <!-- use provided scope because this is not needed at runtime -->
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <version>2.22.0</version>
```

Comment:
Are you sure that you need this specific version of `mockito`? or maybe the good idea will be updating `mockito.version` in root pom file?